src/OptionSettlementEngine.sol

```diff
@@ -19,30 +23,44 @@ import "./TokenURIGenerator.sol";
  * a broad swath of traditional options.
  */
 
-/// @notice This settlement protocol does not support rebase tokens, or fee on transfer tokens
+/// @title A settlement engine for options
+/// @dev This settlement protocol does not support rebase tokens, or fee on transfer tokens
+/// @author 0xAlcibiades
+/// @author Flip-Liquid
+/// @author neodaoist
 contract OptionSettlementEngine is ERC1155, IOptionSettlementEngine {
+    //
```

Comment:
extra //

Reply:
Stylistically I was going to ask about this with you guys. I like it as it gives a buffer underneath the contract definition, bc forge fmt doesn't support a blank line. But I know it's not too standard, I just like it =)